clientconn.go

```diff
@@ -425,7 +424,8 @@ func (cc *ClientConn) validateTransportCredentials() error {
 //
 // Doesn't grab cc.mu as this method is expected to be called only at Dial time.
```

Comment:
docstring needs to be updated to `cc.channelz` and `cc.csMgr.channelz`

Reply:
Done